support/urlUtils.ts

```diff
@@ -157,7 +156,7 @@ export function parseMarker(marker: string): IPromise<Graphic> {
   // ?marker=10406557.402,6590748.134,2526
 
   if (!marker) {
-    return promiseUtils.resolve();
+    return promiseUtils.reject();
```

Comment:
We should be able to import just what we need from `promiseUtils`

Reply:
Good catch. 
I'll make that update.

Reply:
done